worlds/stardew_valley/mods/logic/quests_logic.py

```diff
@@ -88,13 +88,12 @@ def _get_distant_lands_quest_rules(self):
             return {}
 
         return{
-            ModQuest.CorruptedCropsTask: self.logic.region.can_reach(Region.wizard_tower) & self.logic.region.can_reach(Region.witch_swamp) &
-                                         self.logic.has(Fertilizer.quality),
-            ModQuest.WitchOrder: self.logic.region.can_reach(Region.witch_swamp) & self.logic.has(Fertilizer.quality),
-            ModQuest.ANewPot: self.logic.region.can_reach(Region.witch_swamp) & self.logic.region.can_reach(Region.saloon) &
+            ModQuest.CorruptedCropsTask: self.logic.region.can_reach(Region.wizard_tower) & self.logic.has(Fertilizer.deluxe),
```

Comment:
Deluxe fertilizer is a ginger island requirement. Do you want these quests to be ginger island only?

Reply:
I put the requirement on the locations themselves